src/__experimental__/components/numeral-date/numeral-date.style.js

```diff
@@ -0,0 +1,78 @@
+import styled, { css } from 'styled-components';
+import StyledValidationIcon from '../../../components/validations/validation-icon.style';
+import StyledIconSpan from '../input-icon-toggle/input-icon-toggle.style';
+import { baseTheme } from '../../../style/themes';
+import StyledFormField from '../form-field/form-field.style';
+import StyledInput from '../input/input-presentation.style';
+import StyledIcon from '../../../components/icon/icon.style';
+
+const StyledNumeralDate = styled.div`
+  margin: 0px, 0px;
+  display: inline-flex;
+  border: 1px solid transparent;
+  height: 40px;
+  font-size: 14px;
+  font-weight: 400;
+  padding-bottom: 2px;
+  padding-top: 1px; 
+
+  ${StyledFormField} {
+     margin-top: 0px;
+  }
+`;
+
+StyledNumeralDate.defaultProps = { theme: baseTheme };
+export default StyledNumeralDate;
+
```

Comment:
Why `StyledNumeralDate` is a default export and `StyledDateField` is named export?

Reply:
Changed both to named export.